train.py

```diff
@@ -372,6 +376,13 @@ def main():
         optimizer.load_state_dict(optim_state)
         print('Learning rate annealed to: {lr:.6f}'.format(lr=optim_state['param_groups'][0]['lr']))
 
+        if best_wer is None or best_wer > wer:
+            print("Found better validated model, saving to %s" % args.final_model_path)
```

Comment:
Thanks for that, fixed [here](https://github.com/SeanNaren/deepspeech.pytorch/commit/3ffa9a5c2d93000d473208cc5506decc8949f8b3)